xonsh/environ.py

```diff
@@ -580,6 +589,7 @@ def _collapsed_pwd():
     short_cwd=_collapsed_pwd,
     curr_branch=current_branch,
     branch_color=branch_color,
+    current_job=_current_job,
```

Comment:
Can you add this to the relevant `docs/tutorial.rst` section so that people know about it.